kubernetes/is-deployment.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: integration-service-api
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: integration-service-api
+  template:
+    metadata:
+      labels:
+        app: integration-service-api
+    spec:
+      initContainers:
+      - name: wait-for-hornet
+        image: busybox:1.28
+        args:
+        - /bin/sh
+        - -c
+        - >
+          set -x;
+          until nslookup "hornet-tcp.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local";
```

Comment:
It seems like it is always using the `hornet-tcp.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local` as hornet node. As bespoke please use the env var IOTA_HORNET_NODE for lookup